RecoEgamma/EgammaTools/interface/ThreadSafeStringCut.h

```diff
@@ -0,0 +1,41 @@
+#ifndef RecoEgamma_EgammaTools_ThreadSafeStringCut_H
+#define RecoEgamma_EgammaTools_ThreadSafeStringCut_H
+
+#include "CommonTools/Utils/interface/StringObjectFunction.h"
+#include "CommonTools/Utils/interface/StringCutObjectSelector.h"
+
```

Comment:
Oh yes, I should better include it. I will implement and push all comments later, after https://github.com/cms-sw/cmssw/pull/25092 is merged, because then I have to rebase anyway.